src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -141,7 +147,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
         this.updateNativeNotebookContext();
         this.updateNativeNotebookCellContext();
         this.updateMergedContexts();
-        void this.updateDebugContext(e);
+        void this.updateDebugContext(e?.notebook?.getMatchingInterpreter());
```

Comment:
For an async like this we usually use .ignoreErrors() versus putting the void on, just for consistency in this function.